docs/vscode-extension/envchecker-help.md

```diff
@@ -36,6 +36,12 @@ Please refer to [the official website](https://github.com/Azure/azure-functions-
 
 **NOTE**: Please restart all your Visual Studio Code instances after the installation is finished.
 
+## How to install Bicep CLI?
+
+Please refer to [the official website](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/install#install-manually) to install the supported versions: `Bicep CLI v4 `.
```

Comment:
It seems the official document does not mention how to install a specific version of bicep CLI. So how does the users know how to install the v4 version?

Reply:
[https://github.com/Azure/bicep/releases] have all release version